keyboards/ckeys/thedora/keymaps/default/keymap.c

```diff
@@ -0,0 +1,157 @@
+#include QMK_KEYBOARD_H
+
+enum layers {
+  _BASE,                // base layer
+  _MUSIC,               // music mode
+  _MIDI,                // midi mode
+  _MOUSE,               // mouse keys
+  _ADMIN                // admin duties
+};
+
+extern keymap_config_t keymap_config;
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+//           ┌─────────┬─────────┬─────────┬─────────┬─────────┐
+//           │         │         │         │         │         │
+//           │    7    │    8    │    9    │    /    │  ADMIN  │
+//  ROTORY   │         │         │         │         │         │
+//  RIGHT:   ├─────────┼─────────┼─────────┼─────────┼─────────┤
+//  VOL UP   │         │         │         │         │         │
+//           │    4    │    5    │    6    │    *    │  MOUSE  │
+//  ROTORY   │         │         │         │         │         │
+//  LEFT:    ├─────────┼─────────┼─────────┼─────────┼─────────┤
+//  VOL DOWN │         │         │         │         │         │
+//           │    1    │    2    │    3    │    -    │  MIDI   │
+//           │         │         │         │         │         │
+// ┌─────────┼─────────┼─────────┼─────────┼─────────┼─────────┤
+// │  TOGGLE │         │         │         │         │         │
+// │  MUSIC  │    0    │    .    │    =    │    +    │  MUSIC  │
+// │         │         │         │         │         │         │
+// └─────────┴─────────┴─────────┴─────────┴─────────┴─────────┘
+
+  [_BASE] = LAYOUT(
+    KC_KP_7, KC_KP_8,   KC_KP_9,     KC_PSLS, MO(_ADMIN), MU_TOG, \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

Reply:
I had no idea I could remove those. I just kept doing what I've always been doing for the past two years. I agree; much neater without. Removing...